src/Symfony/Component/Console/Command/Command.php

```diff
@@ -572,6 +572,8 @@ public function getSynopsis($short = false)
      * Add a command usage example.
      *
      * @param string $usage The usage, it'll be prefixed with the command name
+     *
+     * @return $this
```

Comment:
that is because of my suggestion I'm afraid
https://github.com/symfony/symfony/pull/19328#discussion_r72576537
sorry if misleading the pr!